test/unit/fixAddress.js

```diff
@@ -0,0 +1,102 @@
+const chai = require('chai');
+
+const fixAddress = require('../../app/lib/fixAddress');
+
+const expect = chai.expect;
+
+describe('Fix Address', () => {
+  it('should gracefully handle undefined address', () => {
+    const address = undefined;
+    fixAddress(address);
```

Comment:
It's unusual to see a test without an expectation. Is it possible to add one (expect to not throw exception maybe) to aid clarity?